Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```diff
@@ -92,7 +92,7 @@
 - type: entity
   parent: DrinkBottleGlassBaseFull
   id: DrinkBlueCuracaoBottleFull
-  name: Miss Blue Curacao
```

Comment:
This sounds like it was intended to be a proper noun, and proper nouns should remain capitalized.